cogs/utilities.py

```diff
@@ -0,0 +1,39 @@
+import discord
+from discord.ext import commands
+import requests
+import json
+import os
+
+class Utilities(commands.Cog):
+	def __init__(self, bot):
+		self.bot = bot
+
+	
+
+	# Report to r/bitcoin mod-log. Subsitute other channel ID's as necessary
+	#todo get channel by ID
+	@commands.command()
```

Comment:
yes this is terrible. until it gets generalized and cleaned up, move messages to some kind of config file maybe, this might belong in its own fork where it currently lives. not sure, your call.